src/main/java/com/monumental/services/MonumentService.java

```diff
@@ -1510,7 +1513,11 @@ public void updateMonumentTags(Monument monument, List<String> newTagNames, bool
             // Un-associate any Tags/Materials from the Monument that were associated previously and no longer are
             for (Tag currentTag : currentTags) {
                 if (!newTagNames.contains(currentTag.getName())) {
-                    this.tagService.removeTagFromMonument(currentTag, monument);
+                    List<MonumentTag> monumentTags = monument.getMonumentTags().stream().filter(mt -> mt.getTag() == currentTag).collect(Collectors.toList());
```

Comment:
Should this list only ever be expecting a single tag?

Reply:
In theory yes, a monument should only ever have one monument tag per tag.

Reply:
can be change it from a list then to a single monument. not super pressed either way